SPECS/xcp-python-libs.spec

```diff
@@ -64,8 +64,7 @@ sed -i "s/dynamic *= *\[\"version\"\]/version = \"%{version}\"/g" pyproject.toml
 %pyproject_install xcp
 %endif
 
-# XCP-ng: FIXME. Needs to be ported to python3 and packaged properly.
-#install -m 0775 %{SOURCE1} %{buildroot}%{python_sitelib}/xcp/updategrub.py
+install -m 0775 %{SOURCE1} %{buildroot}%{python_sitelib}/xcp/updategrub.py
```

Comment:
Doing so bypasses entirely setup.py, which makes the file not to be listed in python package metadata (this is what I meant by saying that we packaged it in a bad way in the past and we should now package it properly).

Reply:
applied, no compat symlink - we'll release all together